api/core/tools/provider/builtin/mingdao_hap/tools/add_worksheet_record.yaml

```diff
@@ -0,0 +1,78 @@
+identity:
+  name: add_worksheet_record
+  author: Ryan Tian
+  label:
+    en_US: Add Worksheet Record
+    zh_Hans: 新增一条工作表记录
+description:
+  human:
+    en_US: Adds a new record to the specified worksheet
+    zh_Hans: 向指定的工作表新增一条记录数据
+  llm: A tool to append a new data entry into a specified worksheet.
+parameters:
+  - name: appkey
+    type: secret-input
+    required: true
+    label:
+      en_US: App Key
+      zh_Hans: App Key
+    human_description:
+      en_US: The AppKey parameter for the HAP application, typically found in the application's API documentation.
+      zh_Hans: HAP 应用的 AppKey 参数，可以从应用 API 文档中查找到
+    llm_description: the AppKey parameter for the HAP application
+    form: form
+
+  - name: sign
```

Comment:
This parameter `sign` should be dynamically calculated as I understand it, does it agree with my understanding here?

Reply:
No, it is static. It will be provided in plaintext as part of the authorization key in each HAP application.